applications/ehr_query_llm/lmm/README.md

```diff
@@ -150,8 +150,53 @@ For the YAML file, the fields needed are:
 
 **publish:** The only important part of this field is the "args" sub-field. This should be a list of your arg names. This is important as this is used as the list of keys to pull the relevant args from the LMM's response, and thus ensure the relevant fields are complete for a given tool use.
 
+For a specific example, please refer to the EHR Agent YAML file below:
+
+```
+description: This tool is used to search the patient's EHR in order to answer questions about the patient, or general questions about the patient's medical history.
+user_prefix: "<|im_start|>user"
+bot_prefix: "<|im_start|>assistant"
+bot_rule_prefix: "<|im_start|>system"
+end_token: "<|im_end|>"
+agent_prompt: |
+  You are NVIDIA's EHR Agent,your job is to assist surgeons as they prepare for surgery.
+  You are an expert when it comes to surgery and medical topics - and answer all questions to the best of your abilities.
+  The patient has signed consent for you to access and discuss all of their electronic records.
+  Be as concise as you can be with your answers.
+
+  You NEVER make-up information that isn't grounded in the provided medical documents.
+
+  If applicable, include the relevant date (use sparingly)
+  The following medical documents may be helpful to answer the surgeon's question:
+  {documents}
+  Use your expert knowledge and the above context to answer the surgeon.
+# This is the request that the LLM replies to, where '{text}' indicates where the transcribed
+# text is inserted into the prompt
+request: "{text}"
+
+ctx_length: 256
+
+grammar: |
+  space ::= " "?
+  string ::= "\"" ([^"\\])* "\"" space 
+  root ::= "{" space "\"name\"" space ":" space "\"EHRAgent\"" space "," space "\"response\"" space ":" space string "}" space
+
+publish:
+  ags:
```

Comment:
Typo: doc describes `publish.args` but this and other YAML files have `publish.ags`.

Could you please help point me to where this field is used? I'm not able to find other matches for `ags` in the code and am wondering if this field is currently respected.